hardware_interface/src/resource_manager.cpp

```diff
@@ -1787,6 +1804,13 @@ bool ResourceManager::state_interface_exists(const std::string & key) const
 
 // END: "used only in tests and locally"
 
+rclcpp::Logger ResourceManager::get_logger() const { return resource_storage_->get_logger(); }
+
+rclcpp::Clock::SharedPtr ResourceManager::get_clock() const
+{
+  return resource_storage_->get_clock();
+}
```

Comment:
Why not to put to `.hpp`?

Reply:
Sure, I can do that

Reply:
I cannot do that because the ´ResourceStorage´ is forward declared in the `.hpp` file and it is showing me

```
error: invalid use of incomplete type ‘class hardware_interface::ResourceStorage’
```